### DISCUSSION TITLE
Blank White Page when accessing the template

### DISCUSSION THREAD

**Discussion by lorvar** [templaza/astroid-framework#786] [General]
Hello everyone.

I'm trying to modify a Templaza Autobike template which uses Astroid , but when I get to the "Template option" it loads a blank white page. So no options to configure anything loads. It only loaded properly once in the past month. I tried changing PHP versions, but nothing changed. I really need to be able to modify the template. Thanks in advance.

**Comment by sonvnn** [comment]
It often relates to server configuration. Can you send me your admin account via sonlv@templaza.com?
I will help you check it.

**Reply by lorvar** [reply]
I can.  Strange thing, I can view it on my Mac but not my Windows 11?

**Comment by Chacapamac** [comment]
• Any error messages in the **browser Inspector** or in the **server logs**?
• Did you try to put on **Joomla debug** or the **debug mode** in the **Astroid plugin**

**Reply by lorvar** [reply]
Chrome Browser
![Screenshot 2024-08-31 105601](https://github.com/user-attachments/assets/5516b57a-b5b0-48da-b72a-00bb7d0ae7fd)

**Reply by sonvnn** [reply]
@lorvar please check if your ssl is working well. Index.js was block because load from an invalid ssl

**Reply by lorvar** [reply]
SSL where, in cpanel or?

**Comment by Chacapamac** [comment]
“Access-Control-Allow-Origin” and “as been blocked by CORE Policy” are the problem

**Explanation**
https://hinty.io/ivictbor/has-been-blocked-by-cors-policy-explain-like-i-am-5/

**Reply by lorvar** [reply]
Sorry I've don't know what this is and never had to adjust anything before. I was asked for access to fix it from @sonvnn and don't know what happened.

**Reply by sonvnn** [reply]
I am on a vacation. I will back to office on Wednesday. But please check your server first. It blocked index.js that is reason the backend can not be loaded.

**Reply by lorvar** [reply]
> I am on a vacation. I will back to office on Wednesday. But please check your server first. It blocked index.js that is reason the backend can not be loaded.

Thank you. I find it strange that I can access it with my phones, tablets and Mac OS, but not in Windows?

**Reply by lorvar** [reply]
Fixed it myself, Sorry couldn't wait.

**Reply by sonvnn** [reply]
@lorvar Thanks for update. I fixed issue "Access-Control-Allow-Origin" on your website. Now you can check it.

**Reply by lorvar** [reply]
Thank you. I had added a prompt in the htaccess to activate it

**Comment by morphico** [comment]
Hi there. I have the same problem. Yesterday everything worked normally but today I have blank page after click "Template options". I have newest framework and newest Template Two files. It's any solutions?

**Reply by morphico** [reply]
Ok, framework reinstalation fixed my problem.
